### PR TITLE
Check SCC/SMT support during self-update

### PR DESCRIPTION
**Pull request by imobachgs**
- [bsc#989787](https://bugzilla.suse.com/show_bug.cgi?id=989787)
- [SCC/SMT support](https://trello.com/c/X3pVQI2X/491-3-sle-12-sp2-p1d-m-when-smt-is-in-use-url-is-converted-to-match-the-smt-for-feature-319716-automatic-update-of-yast-at-beginning) in [FATE#319716](https://fate.suse.com/319716).

### CODE REVIEW

File: aytests/self_update_reg.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e -x
+
+# FATE#319716 - the installer updates need to be installed into inst-sys
+# check that the URL from the registration server is detected.
+zgrep "Updates for 'SLES' are available at.*static/repos.*" /var/log/YaST2/y2log*
```
Comment (schubi2) [inline aytests/self_update_reg.sh]: Hm, be careful with zgrep. See:
https://github.com/yast/aytests-tests/pull/27/files

**Comment by schubi2** [comment]
I would change the zgrep call. Otherwise ----> LGTM

**Comment by imobachgs** [comment]
@schubi2 Ok, fixed now. Thanks!

**Comment by imobachgs** [comment]
This PR should be merged after https://github.com/yast/yast-installation/pull/401 is merged and available in the distribution.

**Pull request merged**